Revert language version fix and find another fix?

In #1541 we updated the language version from 9.0 to 10.0 to workaround a bug with .NET 6 Preview 7; however, this isn't the greatest workaround because it requires everyone building PSES to update their dotnet:

```
CSC : error CS1617: Invalid option '10.0' for /langversion. Use '/langversion:?' to list supported values. [/Users/andschwa/src/PowerShellEditorServices/src/PowerShellEditorServices/PowerShellEditorServices.csproj]
```

We didn't realize this because the CI _always_ comes with an up-to-date dotnet.

The simply solution is for PSES devs to just delete `.dotnet` and let the build script install the latest, but perhaps we should fix it another way.

--- a/PowerShellEditorServices.Common.props
+++ b/PowerShellEditorServices.Common.props
@@ -4,7 +4,7 @@
     <VersionSuffix></VersionSuffix>
     <Company>Microsoft</Company>
     <Copyright>© Microsoft Corporation.</Copyright>
-    <LangVersion>10.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
     <PackageTags>PowerShell;editor;development;language;debugging</PackageTags>
     <PackageLicenseUrl>https://raw.githubusercontent.com/PowerShell/PowerShellEditorServices/master/LICENSE</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
@@ -12,7 +12,9 @@
     <RepositoryUrl>https://github.com/PowerShell/PowerShellEditorServices</RepositoryUrl>
     <DebugType>portable</DebugType>
     <!-- See: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview -->
-    <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <!-- TODO: Enable `EnforceCodeStyleInBuild` -->
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <!-- See: https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/implicit-namespaces -->
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
   </PropertyGroup>
 </Project>

